### PR TITLE
KAZOO-5125: separate descriptions from system_config schemas

### PR DESCRIPTION
**Pull request by fenollp**
This also involves:

* adding a new system_config flag requires the committer to add corresponding documentation
* fixing lots of system_config settings that were read from different places with different default values.
* new way of creating system_config schemas does not merge them with previous version
* removed system_config flags thus get properly cleansed
* fixes to phone_numbers schema

### CODE REVIEW

File: applications/crossbar/priv/couchdb/schemas/system_config.jonny5.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
         },
         "default_{Key}": {
             "description": "jonny5 default {Key}",
-            "type": "integer"
+            "type": "boolean"
```
Comment (jamesaimonetti) [inline applications/crossbar/priv/couchdb/schemas/system_config.jonny5.json]: This should actually be purged from the system config doc. Until we can infer Key, I think it better to leave these off the doc. Thoughts?

File: core/kazoo_apps/src/kapps_speech.hrl [hunk 2]
```diff
@@ -49,7 +49,7 @@
         ,{<<"male/tr-tr">>, <<"eurturkishmale">>}
         ]
        ).
--define(ISPEECH_TTS_URL, kapps_config:get_string(?MOD_CONFIG_CAT, <<"tts_url">>, <<"http://api.ispeech.org/api/json">>)).
+-define(ISPEECH_TTS_URL, kapps_config:get_string(?MOD_CONFIG_CAT, <<"tts_url_ispeech">>, <<"http://api.ispeech.org/api/json">>)).
```
Comment (jamesaimonetti) [inline core/kazoo_apps/src/kapps_speech.hrl]: Are you including a migrate command for this? It is possible that there are other values (I doubt it, but would rather address is than get yelled at that we broke someone's TTS engine).

File: applications/crossbar/priv/couchdb/schemas/system_config.jonny5.json (hunk 1 above)
Comment (fenollp) [inline applications/crossbar/priv/couchdb/schemas/system_config.jonny5.json]: Agreed!

Comment (fenollp) [inline applications/crossbar/priv/couchdb/schemas/system_config.jonny5.json]: So looking into this I'd say I'll do it in another PR.

File: core/kazoo_apps/src/kapps_speech.hrl (hunk 2 above)
Comment (fenollp) [inline core/kazoo_apps/src/kapps_speech.hrl]: Done

File: applications/crossbar/priv/couchdb/schemas/system_config.jonny5.json (hunk 1 above)
Comment (lazedo) [inline applications/crossbar/priv/couchdb/schemas/system_config.jonny5.json]: @fenollp can you remove it now ?

**Pull request merged**